test/integration/storage_versions_test.go

```diff
@@ -23,21 +24,12 @@ import (
 	"k8s.io/kubernetes/pkg/runtime"
 )
 
-// TODO: enable once storage is separable
-// func TestStorageVersionsSeparated(t *testing.T) {
-// 	runStorageTest(t, "separated",
-// 		autoscaling_v1.SchemeGroupVersion,
-// 		batch_v1.SchemeGroupVersion,
-// 		extensions_v1beta1.SchemeGroupVersion,
-// 	)
-// }
-
-func TestStorageVersionsUnified(t *testing.T) {
+func TestStorageVersions(t *testing.T) {
```

Comment:
Yes, down this file: https://github.com/openshift/origin/blob/b7dfdfe5ee48941d156854432738cca7d7eaac2f/test/integration/storage_versions_test.go#L148-L153